cmake/ProjectTcMalloc.cmake

```diff
@@ -0,0 +1,37 @@
+include(ExternalProject)
+
+set(TCMALLOC_CONFIG ./autogen.sh COMMAND ./configure CXXFLAGS=-DHAVE_POSIX_MEMALIGN_SYMBOL=1 --enable-frame-pointers --disable-cpu-profiler --disable-heap-profiler --disable-heap-checker --disable-debugalloc --enable-minimal)
```

Comment:
Please refer here
https://github.com/gperftools/gperftools/blob/fe62a0baab87ba3abca12f4a621532bf67c9a7d2/README#L39

and here 

https://github.com/gperftools/gperftools/blob/fe62a0baab87ba3abca12f4a621532bf67c9a7d2/README#L46-L47

Reply:
--disable-shared

Reply:
why do we need `autogen.sh`? It seems official tutorial doesn't include this step?
https://github.com/gperftools/gperftools/blob/fe62a0baab87ba3abca12f4a621532bf67c9a7d2/INSTALL#L376-L393

Reply:
already add --disable-shared and --disable-test